src/System.Diagnostics.FileVersionInfo/src/Configurations.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <BuildConfigurations>
-      netcore50-Windows_NT;
+      uap-Windows_NT;
```

Comment:
Is the idea to always replace netcore50 configs for uap ones? Or only in the case they don't have a pkgproj like this one?